### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by tsunami-binario**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `main` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `main` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `main`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://docs.github.com/education/manage-coursework-with-github-classroom/leave-feedback-with-pull-requests)”.
</details>

Subscribed: @darklatiz

### CODE REVIEW

File: .replit [hunk 1]
```diff
@@ -1,2 +1,2 @@
 language = "bash"
-run = "bash ejecuta_main.sh"
\ No newline at end of file
+run = "bash ejecuta_pruebas.sh"
```
Comment (tsunami-binario) [inline .replit:2]: No debemos de cambiar este archivo.

File: src/main/java/mx/tsunami/labs/App.java [hunk 2]
```diff
@@ -8,10 +8,12 @@
...
     {
-        System.out.println( "Hello World!" );
+        System.out.println(App.obtenerSaludo("Luis"));
+        System.out.println(App.obtenerSaludo("PEPE"));
+        System.out.println(App.obtenerSaludo("Maria"));
     }
 
     public static String obtenerSaludo(String nombre) {
-        return "";
+        return "Hola " + nombre;
```
Comment (tsunami-binario) [inline src/main/java/mx/tsunami/labs/App.java:17]: Correcto! Muy bien.

File: .replit [hunk 3]
```diff
@@ -1,2 +1,2 @@
 language = "bash"
-run = "bash ejecuta_main.sh"
```
Comment (darklatiz) [inline .replit:2]: que

**Review by tsunami-binario — COMMENTED** [review]
chewck feed back

File: .replit (hunk 1 above)
Comment (darklatiz) [reply .replit:2]: subire un cambio para no modificarlo

File: src/main/java/mx/tsunami/labs/App.java [hunk 4]
```diff
@@ -14,6 +14,6 @@ public static void main( String[] args )
     }
 
     public static String obtenerSaludo(String nombre) {
-        return "Hola " + nombre;
+        return "Hola " + nombre + "!";
```
Comment (tsunami-binario) [inline src/main/java/mx/tsunami/labs/App.java:17]: No es necesario el signo de admiracion, o bien dejame cambiar el unit test para que pueda validar ese tipo de cosas

**Pull request merged**